requirements/common.in

```diff
@@ -0,0 +1,137 @@
+# Packages that are shared between deployment and dev environments.
+gunicorn==20.0.4
+whitenoise[brotli]==5.0.1
```

Comment:
Please note that this file is exactly common.txt, but with hashes removed

Reply:
If we reduce this file to the top level packages (e.g. just django and few more) does the generation of the requirements file become shorter?

Reply:
Yes, as seen in https://github.com/mozilla/treeherder/pull/6141  specifically, there is some Elasticsearch stuff that is no longer needed.